specification/powerbiprivatelinks/resource-manager/Microsoft.PowerBI/stable/2020-06-01/powerbiprivatelinks.json

```diff
@@ -0,0 +1,1627 @@
+        },
+        "privateLinkServiceConnections": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkServiceConnection"
+          },
+          "description": "Specifies the private link service connections of the private endpoint."
+        },
+        "privateLinkServiceProxies": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkServiceProxy"
+          },
+          "description": "Specifies the private link service proxies of the private endpoint."
+        },
+        "connectionDetails": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkConnectionDetail"
+          },
+          "description": "Specifies the connection details of the private endpoint."
+        }
+      }
+    },
+    "PrivateEndpointConnectionProxyResource": {
+      "title": "PrivateEndpointConnectionProxyResource",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Specifies the id of the resource."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Specifies the name of the resource."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Specifies the type of the resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "Specifies the location of the resource."
+        },
+        "etag": {
+          "type": "string",
+          "description": "Specifies the etag of the resource."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PrivateEndpointConnectionProxy",
+          "description": "Specifies the private endpoint proxy resource."
+        }
+      }
+    },
+    "PrivateEndpointConnectionProxy": {
+      "title": "PrivateEndpointConnectionProxy",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Specifies the id of the resource."
+        },
+        "name": {
+          "type": "string",
+          "description": "Specifies the name of the resource."
+        },
+        "type": {
+          "type": "string",
+          "description": "Specifies the type of the resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "Specifies the location of the resource."
+        },
+        "etag": {
+          "type": "string",
+          "description": "Specifies the etag of the resource."
+        },
+        "remotePrivateEndpoint": {
+          "$ref": "#/definitions/RemotePrivateEndpoint",
+          "description": "Specifies the remote private endpoint of the resource."
+        }
+      }
+    },
+    "PrivateEndpointConnection": {
```

Comment:
>PrivateEndpointConnection [](http://example.com/codeflow?start=5&length=25)

Consider $ref of types from privatelinks.json and/or types.json below common-types.

Reply:
will keep the current type.